lib/src/components/navigation bar/navigation_bar.dart

```diff
@@ -278,11 +293,9 @@ class _NavigationItem extends ZetaStatelessWidget {
                 ),
                 SizedBox(height: Zeta.of(context).spacing.small),
                 if (item.label != null)
-                  ExcludeSemantics(
-                    child: Text(
-                      item.label!,
-                      style: Theme.of(context).textTheme.labelSmall?.copyWith(color: elementColor),
-                    ),
+                  Text(
```

Comment:
This is linked to the above - we want a badge or any other widget to be individual, but the text is part of the parent so we need to exclude it so that it is not duplicated

Reply:
So wrap this text in excludeSemantics

Reply:
yes please